package.json

```diff
@@ -165,6 +163,7 @@
 		"@babel/preset-react": "^7.10.4",
 		"@componentdriven/csf": "0.0.2-alpha.0",
 		"@jackfranklin/test-data-bot": "^1.3.0",
+		"@popperjs/core": "^2.9.2",
```

Comment:
Note for the merge reviewer; this package is needed for the storybook. It was being provided by `@wordpress/components`